model/error/event_test.go

```diff
@@ -292,6 +292,99 @@ func TestErrorEventDecode(t *testing.T) {
 	}
 }
 
+func TestHandleExceptionTree(t *testing.T) {
+	errorEvent := map[string]interface{}{
+		"exception": map[string]interface{}{
+			"message": "message0",
+			"type":    "type0",
+			"stacktrace": []interface{}{
+				map[string]interface{}{
+					"filename": "file0",
+				},
+			},
+			"cause": []interface{}{
+				map[string]interface{}{"message": "message1", "type": "type1"},
+				map[string]interface{}{"message": "message2", "type": "type2", "parent": json.Number("0"), "cause": []interface{}{
+					map[string]interface{}{"message": "message3", "type": "type3", "cause": []interface{}{
+						map[string]interface{}{"message": "message4", "type": "type4"},
+						map[string]interface{}{"message": "message5", "type": "type5", "parent": json.Number("3")},
+					}},
+				}},
+				map[string]interface{}{"message": "message6", "type": "type6", "parent": json.Number("0")},
+			},
+		},
+	}
+	result, err := DecodeEvent(errorEvent, m.Config{}, nil)
+	require.NoError(t, err)
+
+	event := result.(*Event)
+	exceptions := flattenExceptionChain(event.Exception)
+
+	assert.Equal(t, len(exceptions), 7)
```

Comment:
`assert.Equal(t, 7, len(exceptions)`, the signature is `func Equal(t TestingT, expected, actual interface{}, ....`